### DISCUSSION TITLE
Produkty z poza TecDoca a baza pojazdów

### DISCUSSION THREAD

**Discussion by zulus** [allegro/allegro-api#4483] [Suggestions - rejected]
:bulb: **Sugestia**
Mamy sporo produktów (głównie marki własne, ale też niektóre zamienniki) które mają już potworzone produkty w allegro, ale bez podpiętej bazy pojazdów. Niestety w związki z tym musieliśmy wyciąć produktyzacje np dla marki STARLINE.

Dobrze by było, gdyby jeśli produkt niema wprost listy pasujących aut, żeby jednak oferta mogła dopiąć własne.

**Comment by MartaNowaczyk** [comment]
Dziękuję za sugestię. Zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przenoszę wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions).

**Comment by MartaNowaczyk** [comment]
Jeśli produkt nie ma swojej własnej listy "Pasuje do", można podpiąć ją manualnie.
